### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by mchangrh**
To test this pull request, follow the [instructions in the wiki](https://github.com/ajayyy/SponsorBlock/wiki/Testing-a-Pull-Request).

***

I just got really distracted and now I have a messy working codebase

- [x] Replace String.substr with String.slice
- [x] Fix misc eslint errors
- [x] Fix mobile ChannelID lookup #1153 
- [x] Fix taking audio focus #1053 
  - set mediaSession metadata to null and then set to old metadata

### CODE REVIEW

File: src/content.ts [hunk 1]
```diff
@@ -971,8 +967,7 @@ function getYouTubeVideoIDFromURL(url: string): string | boolean {
         return id.length == 11 ? id : false;
     } else if (urlObject.pathname.startsWith("/embed/") || urlObject.pathname.startsWith("/shorts/")) {
         try {
-            const id = urlObject.pathname.split("/")[2];
-            if (id && id.length >= 11) return id.substr(0, 11);
+            return urlObject.pathname.split("/")[2].slice(0, 11);
```
Comment (ajayyy) [inline src/content.ts:970]: Why this change?

Comment (mchangrh) [reply src/content.ts:970]: slice won't return a different value if it's already under the length, adding the check for id and id.length isn't necessary

Comment (mchangrh) [reply src/content.ts:970]: ah I see what you mean

Comment (mchangrh) [reply src/content.ts]: reverted
